src/main/java/views/session_load/SessionLoadMenuV.java

```diff
@@ -0,0 +1,84 @@
+package views.session_load;
+
+import use_cases.session_load.SessionLoadException;
+import use_cases.session_load.SessionLoadOD;
+import views.main_menu.MainMenuV;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SessionLoadMenuV extends JPanel implements SessionLoadMenuVB, ActionListener {
+    private final SessionLoadC controller;
+    private final MainMenuV mainMenuV;
+
+
+    public SessionLoadMenuV(SessionLoadC controller, MainMenuV mainMenuV) {
+        this.controller = controller;
+        this.mainMenuV = mainMenuV;
+
+        JLabel title = new JLabel("Load session");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JPanel buttons = new JPanel();
+        JButton load_new = new JButton("Create new session");
+        JButton load_file = new JButton("Load session from a file");
+
+        load_new.addActionListener(this);
+        load_file.addActionListener(this);
+
+        buttons.add(load_new);
+        buttons.add(load_file);
+
+
+        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        this.add(title);
+        this.add(buttons);
+    }
+
+    /**
+     * Displays an error message in a popup
+     *
+     * @param message an error message to display
+     */
+    @Override
+    public void displayError(String message) {
+        JOptionPane.showMessageDialog(this, message);
+    }
+
+    /**
+     * Closes this menu by setting its visibility to false
+     */
+    @Override
+    public void close() {
+        this.setVisible(false);
+    }
```

Comment:
I'm not sure why do you need this method. It's just one line either way.

Reply:
Oh, cause its part of the output boundary, I guess I could write a method called displaySuccess in the output boundary that does both this and open the main menu, but I kinda like the readability this provides in the actionPerformed method below where you can see exactly what it is doing by first closing this menu and then opening it, because I feel like this.close() + this.mainMenuV.openMainMenu() is more descriptive even if its a bit more verbose

Reply:
If you feel strongly towards changing it I can change it no problem though

Reply:
Yeah, I'm still not sure how much difference it's going to make. I feel like it's just redundant code, and being more descriptive just made it harder to understand what it's doing, compared to this.setVisible(false).

Reply:
Yeah after thinking about it a bit more I see that you were right now. I made a new displaySuccess method just like the displayError in the view boundary and implemented it so that it does both this.setVisible(false) and opens the main menu. The actionPerformed method is simpler now and it's clearer in displaySuccess what it does. Do you think this was a good solution?

Reply:
Yes, it looks good!